frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/Toolbar/DesktopToolbar.module.css

```
@@ -1,5 +1,16 @@
+@media screen and (max-width: 767px) {
+  .root {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .root {
+    display: inline-flex;
+  }
+}
+
```

Q: Doesn't it working? 

```suggestion
.root {
  display: none;
}

@media screen and (min-width: 768px) {
  .root {
    display: inline-flex;
  }
}
```

---

Oh, I see! It does work.   :) Thank you! @MH4GF 
It looks like we need to be careful about the order when fixing this. Is that alright?  
Around line 13, `.root` is defined again, but it seems necessary to define `.root` twice.

---

@hoshinotsuyoshi 
That's true!! This suggestion can't be accepted as is.

---

fixed. e5815794a859d how about this? @MH4GF

---

nice!!!